services/web-app/utils/string.js

```diff
@@ -46,3 +46,13 @@ export const isJsonString = (str) => {
   }
   return true
 }
+
+/**
+ * Adds https protocol to url if no protocol present
+ * @param {string} url
+ * @returns {string} url with protocol
+ */
+export const getUrlWithProtocol = (url) => {
```

Comment:
@francinelucca do we need to `(url = "")` initialize with an empty string so if nothing gets passed in, the `startsWith()` doesn't throw an error?

Reply:
Done!